### PR TITLE
fixes #35466 - change repo location of puppet-release files

### PR DESCRIPTION
**Pull request by ikonia**

### CODE REVIEW

**Comment by evgeni** [comment]
The Katello failures are unrelated and this looks good to me.

@ekohl you had reviewed #9396 before -- mind having a final look here?

File: app/views/unattended/provisioning_templates/snippet/puppetlabs_repo.erb [hunk 1]
```diff
@@ -41,18 +41,15 @@ if host_param_true?('enable-puppetlabs-repo')
   repo_subdir = ''
```
Comment (sbernhard) [inline app/views/unattended/provisioning_templates/snippet/puppetlabs_repo.erb]: remove this repo_subdir, too?

Comment (ikonia) [reply app/views/unattended/provisioning_templates/snippet/puppetlabs_repo.erb]: done - I was going to leave that for when I pulled the whole puppet5 release from the templates as it looked like the generic puppetlabs-release may have had some depends on a a subdir, but as puppet5 is EOL I guess it doesn't matter, and it looks like that may have been incorrect when I first looked it up due to symlinks

**Comment by sbernhard** [comment]
Please rebase & squash so that you only have one commit.

**Comment by ikonia** [comment]
> Please rebase & squash so that you only have one commit.

I did git rebase, and picked the single additional commit rather than squash, hence why there is now two commits when there should have been one.

**Review by ekohl — APPROVED** [review]
Katello failure unrelated. Thanks!

**Pull request merged**